README.md

```diff
@@ -17,73 +17,75 @@ You can cycle through the fonts by pressing the `F10` key, and cycle backwards w
 
 You need to set the `fonts_list` key in the `Preferences.sublime-settings` file to a list of fonts you want to cycle within. The fonts need to be **pre-installed** on the system.
 
-You can specify just a font name or an object with the properties `font_face`, `font_size`, `line_padding_bottom` and `line_padding_top`. `font_face` is a required property on the object.
+You can specify just a font name or an object with the properties `font_face`, `font_size`, `line_padding_bottom`, `line_padding_top`, `word_wrap` and `wrap_width`. `font_face` is a required property on the object.
 
 ```JSON
 {
-  "fonts_list":
-    [
-      "Andale Mono",
-      "Consolas",
-      "Courier",
-      "Courier New",
-      "Envy Code R",
-      "Inconsolata",
-      "Lucida Console",
-      "Menlo",
-      "Meslo LG S",
-      "Meslo LG M",
-      "Meslo LG L",
-      "Monaco",
-      "ProFontX",
-      "Source Code Pro",
-      "Ubuntu Mono"
-    ]
+	"fonts_list":
+		[
+			"Andale Mono",
+			"Consolas",
+			"Courier",
+			"Courier New",
+			"Envy Code R",
+			"Inconsolata",
+			"Lucida Console",
+			"Menlo",
+			"Meslo LG S",
+			"Meslo LG M",
+			"Meslo LG L",
+			"Monaco",
+			"ProFontX",
+			"Source Code Pro",
+			"Ubuntu Mono"
+		]
```

Comment:
Can you undo this whitespace change? Same with a couple more below.

Reply:
Done! I fixed the whitespace change, added the updated PackageControl URL and also added some examples with proportional width fonts. Sorry if it's a little messy, I've never done a git push/pull before.